client/src/components/ToolsTab.tsx

```diff
@@ -40,7 +40,13 @@ const ToolsTab = ({
 }) => {
   const [params, setParams] = useState<Record<string, unknown>>({});
   useEffect(() => {
-    setParams({});
+    const params = Object.entries(
+      selectedTool?.inputSchema.properties ?? [],
+    ).map(([key, value]) => [
+      key,
+      generateDefaultValue(value as JsonSchemaType),
+    ]);
+    setParams(Object.fromEntries(params));
```

Comment:
This fixes a separate bug where all params are initialized to `undefined`, which causes errors if someone hits `Run Tool` immediately and never interacts with the input.